### ISSUE TITLE
"Post from URL" doesn't work with MP4 urls

### ISSUE THREAD

**Issue by LeeroyGarfield** [TuxedoTako/4chan-xt#127]
Even though this new update fixes the unsupported MP4 upload, the option to upload from URL doesn't work.

**Comment by TuxedoTako** [comment]
It works when I test it. Can you provide the URL that fails?

[msedge_W3wTTgE4bh.webm](https://github.com/user-attachments/assets/060ee21f-f866-4655-bf90-795d72855866)

**Comment by LeeroyGarfield** [comment]
Huh, So I tested it with a Hololive MP4 from Tenor, and it actually works, but for some reason, the audioless MP4 from [CCWeek](https://ccweek.otomos.me/) does the opposite, as you can see here:
https://github.com/ccweek-img/ccweek-images/raw/refs/heads/main/videos/659841215.mp4
https://github.com/ccweek-img/ccweek-images/raw/refs/heads/main/videos/8541621981.mp4
Weirdly enough, if I download it from the same links and then upload them, they work just fine.

Sorry, about this. I noticed this during CC's stream but I didn't dig into it further. Thanks for all your hard work.

**Comment by TuxedoTako** [comment]
Looks like the server reports the file type as `application/octet-stream`. I also noticed that oping those URL's in the browser result in the download prompt instead of playing the video in the browser.

![msedge_uBMquQDrSk](https://github.com/user-attachments/assets/c2b4b02e-7665-4702-83bf-e7bf71eefb24)

**Comment by LeeroyGarfield** [comment]
Yeah, the download prompt is probably the reason why it doesn't work.
